src/handlers/aesEncryption.js

```diff
@@ -0,0 +1,50 @@
+import { NativeModules } from 'react-native';
+const AesEncryption = NativeModules.Aes;
+
+export default class AesEncryptor {
```

Comment:
maybe it will be a bit more clear to make it an object with global variable `key` since it's used only once in a global namespace?

Reply:
Didn't need the key variable. Thanks for the catch!